javascript/packages/cli/src/actions/setup.ts

```diff
@@ -0,0 +1,235 @@
+import { askQuestion, convertBytes, decorators } from "@zombienet/utils";
+import cliProgress from "cli-progress";
+import fs from "fs";
+import path from "path";
+
+interface OptIf {
+  [key: string]: { name: string; url?: string; size?: string };
+}
+
+const options: OptIf = {};
+/**
+ * Setup - easily download latest artifacts and make them executablein order to use them with zombienet
+ * Read more here: https://paritytech.github.io/zombienet/cli/setup.html
+ * @param params binaries that willbe downloaded and set up. Possible values: `polkadot` `polkadot-parachain`
+ * @returns
+ */
+export async function setup(params: any) {
+  const POSSIBLE_BINARIES = ["polkadot", "polkadot-parachain"];
+
+  console.log(decorators.green("\n\n🧟🧟🧟 ZombieNet Setup 🧟🧟🧟\n\n"));
```

Comment:
Have you heard about the [colors](https://www.npmjs.com/package/colors) library ? 

It could enable to simplify the code like this (it extends the String.constructor):
```suggestion
  console.log("\n\n🧟🧟🧟 ZombieNet Setup 🧟🧟🧟\n\n".green);
```

Simpler and better because we don't have to manage our own "colors" system in utils.